### ISSUE TITLE
False positive on Lint/InterpolationCheck

### ISSUE THREAD

**Issue by kddeisz** [bbatsov/rubocop#4730]
## Expected behavior

Lint/InterpolationCheck should function appropriately for double quoted strings.

## Actual behavior

I'm not entirely sure why it's triggering for this.

## Steps to reproduce the problem

```ruby
variable = 5
"alpha #{variable} beta \#{gamma}\" delta"
```

leads to

```
Inspecting 1 file
W

Offenses:

example.rb:2:19: W: Lint/InterpolationCheck: Interpolation in single quoted string detected. Use double quoted strings if you need interpolation.
"alpha #{variable} beta \#{gamma}\" delta"
                  ^^^^^^^^^^^^^^^^^^^^^^^

1 file inspected, 1 offense detected
```

## RuboCop version

```
$ rubocop -V
0.50.0 (using Parser 2.4.0.0, running on ruby 2.4.1 x86_64-darwin16)
```
